contracts/ColonyTask.sol

```diff
@@ -297,22 +297,22 @@ contract ColonyTask is ColonyStorage {
     return taskWorkRatings[_id].secret[_role];
   }
 
-  function setTaskManagerRole(uint256 _id, address _user) public
+  function setTaskManagerRole(uint256 _id, address payable _user) public
   stoppable
   self()
   isAdmin(_user)
   {
     setTaskRoleUser(_id, TaskRole.Manager, _user);
   }
 
-  function setTaskEvaluatorRole(uint256 _id, address _user) public stoppable self {
+  function setTaskEvaluatorRole(uint256 _id, address payable _user) public stoppable self {
```

Comment:
Just so we don't forget what we discussed, let's bring these into the IColony interface.